examples/README.md

```diff
@@ -10,6 +10,7 @@ This folder contains example scripts showing how to use Node Redis in different
 | `connect-as-acl-user.js`    | Connect to Redis 6 using an ACL user                                                                                                                |
 | `count-min-sketch.js`       | Estimate the frequency of a given event using the [RedisBloom](https://redisbloom.io) Count-Min Sketch                                              |
 | `cuckoo-filter.js`          | Space efficient set membership checks with a [Cuckoo Filter](https://en.wikipedia.org/wiki/Cuckoo_filter) using [RedisBloom](https://redisbloom.io) |
+| `get-server-time.js`        | Get the time from the Redis server                                                                                                                  |
```

Comment:
Is the time a current UNIX time like `now()`? so we can have clearer description.

Reply:
No need, it's shown as a comment in the code and documented at https://redis.io/commands/time